contents/tutorials/limit-session-recordings.md

```diff
@@ -162,6 +172,30 @@ This is useful if you want to record specific parts or paths on a page such as:
 - new or updated features
 - smaller, specific components within a larger component.
 
+## Sampling
+
+You can configure sampling to limit the number of sessions you record for each user. This is useful if you want to record a percentage of sessions for all users. Sampling helps reduce the number of sessions you record, but it doesn’t let you control which sessions are included.
+
+Our recommendation is to start with no sampling (e.g. 100% - the default) or a low sampling rate (e.g. 90% or 95%) and increase it as needed. This helps you get a sense of how many sessions you’re recording and how much data you’re collecting.
```

Comment:
> low sampling rate (e.g. 90% or 95%)

should it be:

> high sampling rate and decrease it as needed

instead?

Reply:
High sampling rates capture more data points, and low sampling rate captures less data points.
In this sentence, 90-95% would be sampling 90-95% of all sessions, so its more data points, hence a high sampling rate

Reply:
Ah, yeah, that's totally right 🙈